admin/services/data-access/pupil-restart.data.service.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const PupilRestart = require('../../models/pupil-restart')
+const pupilRestartDataService = {}
+
+/**
+ * Create a new Pupil restart
+ * @param data
+ * @return {Promise}
+ */
+pupilRestartDataService.create = async function (data) {
+  const pupilRestart = new PupilRestart(data)
+  return pupilRestart.save()
```

Comment:
Actually won't this return the mongoose model?

Should it actually be:
```
await pupilRestart.save()
return pupilRestart.toObject()
```

Reply:
@jon-shipley updated